src/daemons/rhsmcertd.c

```diff
@@ -107,6 +109,9 @@ static GOptionEntry entries[] = {
     {"auto-registration-interval", 'r', 0, G_OPTION_ARG_INT, &arg_reg_interval_minutes,
             N_("interval to run auto-registration (in minutes)"),
             "MINUTES"},
+    {"disable-auto-attach", 'x', 0, G_OPTION_ARG_NONE, &arg_reg_interval_minutes,
+            N_("Disable auto attach on successful auto registration (requires -a)."),
```

Comment:
Since it's specifically for this use case. We want to disable auto attach in a persistent way that is still compatible with future runs of rhsmcertd and changes made on the server. We don't want rhsmcertd to never auto attach again if the customer runs 'subscription-manager auto-attach --enable' or similar from the server side.